spynnaker/pyNN/models/neural_projections/connectors/from_list_connector.py

```diff
@@ -215,7 +215,7 @@ def create_synaptic_block(
             return numpy.zeros(0, dtype=self.NUMPY_SYNAPSES_DTYPE)
         self._split_connections(pre_slices, post_slices)
         indices = self.__split_conn_list[
-            (pre_vertex_slice.hi_atom, post_vertex_slice.hi_atom)]
+            pre_vertex_slice.hi_atom, post_vertex_slice.hi_atom]
```

Comment:
Does this have any effect on the speed (yes, I know this is a FromList...)?

Reply:
No. Python forms the tuple when you put a comma-separated sequence inside brackets. I just don't like having the extra parens in there.